content/about/introduction/introduction.html

```diff
@@ -0,0 +1,171 @@
+          </li>
+          <li>
+            Use ARIA, HTML, and CSS to implement fundamental accessibility practices, such as providing accessible names, enabling keyboard interaction, conveying page structure to assistive technologies, and supporting high contrast operating system settings.
+          </li>
+        </ul>
+        <p>
+          The primary aim of the APG is to increase the availability of robust accessible experiences by building broad understanding  and encouraging adoption of web accessibility practices that:
+        </p>
+        <ul>
+          <li>Are consistent with the intent of ARIA, HTML, CSS, and related  specifications.</li>
+          <li>Support assistive technology interoperability, which is assessed by  the <a href="https://aria-at.w3.org/">ARIA and Assistive Technologies Project</a>.</li>
+          <li>Promote practical techniques for developing modern GUIs that both support usability for people with disabilities and conform with <a href="https://www.w3.org/TR/WCAG21/">WCAG</a> requirements.</li>
+        </ul>
+      </section>
+
+      <section id="apg-is--not-normative">
+        <h2>APG is Not a Normative Standard</h2>
+        <p>
+          The accessibility guidance in the APG is different from accessibility requirements specified by <a href="https://www.w3.org/TR/WCAG21/">WCAG</a> and <a class="specref" href="#abstract">ARIA</a>.
+          WCAG and ARIA are "W3C normative technical standards" while the APG is an "informative" resource.
+          W3C normative technical standards, which are formally known as "W3C Recommendations", have conformance models that define how to comply with the requirements documented in the standard.
+          Those requirements are developed using a rigorous process defined by
+          The <a href="https://www.w3.org/Consortium/Process#rec-track">W3C Recommendation Track</a>.
+        </p>
+        <p>
+          The APG does not specify normative requirements and thus does not have a conformance model.
+          The <a href="https://www.w3.org/groups/tf/aria-practices">ARIA Authoring Practices Task Force</a>, which is part of the <a href="https://www.w3.org/groups/wg/aria">ARIA Working Group</a>,
+          uses consensus to ensure APG guidance is consistent with all relevant specifications, including ARIA, but its process for doing so is not bound by the W3C Recommendation Track process.
+        </p>
+        <p>
+          Some guidance provided by the APG is not covered by any normative specification.
+          Most notably, the patterns include keyboard interaction models that reflect a combination of one normative requirement specified by ARIA and conventions established by popular operating systems and software.
+          That is, in the section on <a href="#managingfocus">Managing focus</a>, ARIA specifies that the <kbd>Tab</kbd> key should not move focus inside of certain types of widgets.
```

Comment:
The "Managing Focus" link points to an ID `managingfocus` that doesn't exist in the page